src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        class="card__image image"
+        src="./images/imac.jpeg"
+        alt="apple desktop computer">
+
+      <p class="card__title title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="card__code code">Product code: 195434</p>
+
+      <div class="card__review review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="smallFont">Reviews: 5</p>
+      </div>
+
+      <div class="card__cost cost">
+        <p class="mediumFont">Price:</p>
+        <p class="largeFont">$2,199</p>
+      </div>
+
+      <a
+        href="#"
+        class="card__button button"
+        data-qa="hover">
+        BUY
```

Comment:
Add uppercase using css
```suggestion
      <a
        href="#"
        class="card__button button"
        data-qa="hover"
      >
        buy
```